src/main/resources/data/immersiveengineering/loot_tables/blocks/grass_drops.json

```diff
@@ -0,0 +1,43 @@
+{
+	"type": "minecraft:block",
+	"pools": [
+		{
+			"name": "grass_drops",
```

Comment:
This pool-name does not match the name used in the event.

Reply:
True. Consistent now.